helm/polaris/values.yaml

```diff
@@ -0,0 +1,391 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# -- The number of replicas to deploy (horizontal scaling).
+# Beware that replicas are stateless; don't set this number > 1 when using in-memory meta store manager.
+replicaCount: 1
+
+image:
+  # -- The image repository to pull from.
+  repository: localhost:5001/polaris
+  # -- The image pull policy.
+  pullPolicy: IfNotPresent
+  # -- The image tag.
+  tag: "latest"
+
+toolsImage:
+  # -- The image repository to pull from (must have jar binary included).
+  repository: registry.access.redhat.com/ubi9/openjdk-21
+  # -- The image pull policy.
+  pullPolicy: IfNotPresent
+  # -- The image tag.
+  tag: "latest"
+
+# -- References to secrets in the same namespace to use for pulling any of the images used by this
+# chart. Each entry is a LocalObjectReference to an existing secret in the namespace. The secret
+# must contain a .dockerconfigjson key with a base64-encoded Docker configuration file. See
+# https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/ for more
+# information.
+imagePullSecrets: []
+#  - name: registry-creds
+
+
+serviceAccount:
+  # -- Specifies whether a service account should be created.
+  create: true
+  # -- Annotations to add to the service account.
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template.
+  name: ""
+
+
+# -- Annotations to apply to polaris pods.
+podAnnotations: {}
+
+# -- Additional Labels to apply to polaris pods.
+podLabels: {}
+
+# -- Additional Labels to apply to polaris configmap.
+configMapLabels: {}
+
+# -- Security context for the polaris pod. See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/.
+podSecurityContext:
+  {}
+  # fsGroup: 2000
+
+# -- Security context for the polaris container. See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/.
+securityContext:
+  {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+# Polaris service settings.
+service:
+  # -- The type of service to create.
+  type: ClusterIP
+  # -- The ports the service will listen on. Two ports are required: one for the Polaris service and
+  # one for the metrics API.
+  # Note: port names must be unique and no more than 15 characters long.
+  ports:
+    # polaris-server: The port the Polaris server listens on for API requests.
+    polaris-service: 8181
+    # polaris-mgmt: The port the Polaris server listens on for metrics API requests (health checks, metrics, etc.).
+    polaris-metrics: 8182
```

Comment:
Should this be `polaris-mgmt` as implied by the comment above?

Reply:
Good catch here. Renamed to `polaris-metrics` in both places. This is because for Nessie, the management API does more than just metrics. Then on Polaris side, 8182 only has metrics as of now. See naming discussion in https://github.com/apache/polaris/pull/127.